keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -260,6 +262,12 @@ nitty-gritty.
   The cluster name should be consistent for the life of a cluster and its
   membership in the clusterset. Implementations should treat name mutation as
   a delete of the membership followed by recreation with the new name.
+- **cluster network** - An identifier for the cluster network. Each cluster can have an optional name that can identify the network its running in. The network name must be a valid [RFC
```

Comment:
This makes an implicit assumption that egress network connectivity is mapped via identifiers to figure if given traffic needs to go through a proxy or is a direct connection. Has it been considered whether or not to express that specifically (e.g. the API is focused on "this traffic is proxied" vs saying "we have labeled our networks and this labeling implicitly implies that there is some proxying". The second statement assumes more about the environment and imposes a specific naming, labeling mechanism.

Reply:
Yes, will reword this to more generic after the right name is chosen in favor of `network`